front/src/modules/pathfinding/helpers/__tests__/reversePathSteps.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('reversePathSteps', () => {
       positionOnPath: 10402000,
       name: 'Pont-Hébert',
       kp: '10+396',
-      arrival: new Duration(1500000),
+      arrival: new Duration({ milliseconds: 1500000 }),
```

Comment:
I guess, for readability, we can favor using seconds or minutes now that the constructor allows it

Reply:
I decided to keep the previous number to make it easier to check that the value is still the same, and because readability doesn't matter very much here since it's a bunch of arbitrary numbers. I can change it if you prefer :P

Reply:
fine to leave them too